src/rust/src/rstringcache.rs

```diff
@@ -1,42 +1,15 @@
-use crate::robj_to;
-use crate::rpolarserr::*;
 use extendr_api::prelude::*;
 
-// public literal switch
-// if toggle==True & counter.is_zero(), set counter to one
-// if toggle==False & counter.is_not_zero, set counter to zero
-// otherwise do leave counter as is
 #[extendr]
-pub fn enable_string_cache(toggle: Robj) -> RResult<()> {
-    let toggle = robj_to!(bool, toggle)?;
-    match (toggle, polars_core::using_string_cache()) {
-        (true, false) => polars_core::enable_string_cache(true), //counter 0 -> 1
-        (false, true) => {
-            // counter x -> 0
-            while polars_core::using_string_cache() {
-                polars_core::enable_string_cache(false)
-            }
-        }
-        _ => (), // switch already as requested
-    };
-    Ok(())
+pub fn enable_string_cache() {
+    polars_core::enable_string_cache();
```

Comment:
Just following py-polars here. Now that we have `enable_` and `disable_string_cache()`, we don't need this `match ` anymore